src/condor_gridmanager/remote_gahp

```diff
@@ -44,6 +46,7 @@ Options: \n \
  --rgahp-nopass \t\tno passphrase protecting ssh private key (same as empty rgahp-pass)\n \
  --rgahp-glite PATH \tpath to the directory of the script (~/bosco/glite)\n \
  --rgahp-script SCRIPT \tpath to script to start up blahp (PATH/bin/batch_gahp)\n \
+ --rgahp-nobatchmode \t\tDisable SSH BatchMode (yes)\n \
```

Comment:
Unrelated to your change, but -- wow, this usage string is something else.

Reply:
Yeah, I was just blindly copying previous usage strings but it doesn't make sense to tell the user what the default is for a flag. Hopefully it's more sensible now.